### ISSUE TITLE
Drawer Hack

### ISSUE THREAD

**Issue by ahmedkassem** [dart-flitter/flutter_webview_plugin#113]
can we assign the hidden attribute to the state and create a function to toggle between hidden = true and hidden = false , to hide the webview rect when the drawer is opened
and we use the leading in Appbar to control that

State and function to toggle visibility
`
bool _visibility = false;
  _handleDrawer(){
      _scaffoldKey.currentState.openDrawer();
           setState(() {
          _visibility = true;
        });
  }
`

`      appBar: new AppBar(
        centerTitle: true,
        leading: new IconButton(icon: new Icon(
          Icons.menu
        ),onPressed:_handleDrawer,),
        title: new Text('Plugin example app'),
      ),
`

**Comment by zoechi** [comment]
Isn't this what #86 added? (didn't have a closer look myself)

**Comment by gedw99** [comment]
Yeah #86 looks like it allows toggling the webview being hidden OR displayed.

But it's going to be clunky because the transition will look weird. You hide the webview and the screen goes white and the drawer starts to slide across from left to right. 
What is needed maybe is the ability to fade the webview out as the drawer opens. It might be enough to do the trick.
Or you could take a screenshot of the webview and put that on after you hide the webview and then slide the drawer across.
Then when the drawer has fully closed you turn off the screenshot and turn on the webview.

What do you think ?

**Comment by ahmedkassem** [comment]
what is the function to show and hide the webview because  i tried these and they are not right : 
`    flutterWebviewPlugin.hide();
`
`    flutterWebviewPlugin.show();
`

**Comment by lejard-h** [comment]
@gedw99 I thought about taking a screenshot but never had the chance to test it and I don't know if it is possible.

@ahmedkassem try to upgrade the package, functions you are trying to use are correct

**Comment by ahmedkassem** [comment]
@lejard-h i will give it a try , but can you help with the function when the drawer is open and the user clicks on the screen itself not the drawer (on drawer dismiss i want to fire the flutterWebviewPlugin.hide(); function)
